src/development/LocalTest/Services/Authorization/Implementation/RolesWrapper.cs

```diff
@@ -46,7 +46,7 @@ public async Task<List<Role>> GetDecisionPointRolesForUser(int coveredByUserId,
         private string GetRolesPath(int userId, int resourcePartyId)
         {
             string unitTestFolder = this._localPlatformSettings.LocalTestingStaticTestDataPath + this._localPlatformSettings.AuthorizationDataFolder + this._localPlatformSettings.RolesFolder;
-            return Path.Combine(unitTestFolder, @"User_" + userId + @"\party_" + resourcePartyId + @"\roles.json");
+            return Path.Combine(unitTestFolder, @"User_" + userId + @"/party_" + resourcePartyId + @"/roles.json");
```

Comment:
Why not 
```c#
Path.Combine(unitTestFolder, @"User_" + userId, @"party_" + resourcePartyId, @"roles.json");
```

Reply:
@alt-how Yes, that will work. The only thing is that the path on windows will be inconsistent, since `unitTestFolder` is build up using `/` as path separator, while Path.Combine will use `\` on Windows unless something else is specified. So the result would be something like
`"C:/Repos/altinn-studio/src/development/TestData/authorization/roles/User_12345\\party_12345\\roles.json"`. 
This works, but looks a little strange when debugging.

Reply:
Ok. Maybe drop the initial Path.Combine as well then, if .net core is able to understand '/' on all platforms? '@' can also be dropped since we don't need escaping.

Reply:
We consistently use `/` at the end of all our paths, f.ex. `LocalPlatformSettings` - I think it's ok to do that here as well. I also think it's a good idea to keep Path.Combine, f.ex. if some of the path settings are changed. The code could definitely be cleaner though, I pushed a new version now. What do you think @alt-how ?